crates/ruff/src/rules/flake8_bugbear/rules/mutable_argument_default.rs

```diff
@@ -49,6 +50,9 @@ use crate::registry::AsRule;
 /// l2 = add_to_list(1)  # [1]
 /// ```
 ///
+/// ## Options
+/// - `flake8-bugbear.extend-immutable-calls`
```

Comment:
Should we mention the effect of this in the rule description above?

Reply:
I didn't really see precedence for this elsewhere but I didn't look very hard! I'm happy to mention it

Reply:
We do it in some places, grep for "[`flake8-", but totally up to you.